resources/jaeger/README.md

```diff
@@ -6,12 +6,7 @@
 ## Details
```

Comment:
We can even remove this part (details), since it's in Jaeger documentation and link to the official doc.